packages/infra/lib/ihe-gateway-v2-stack.ts

```diff
@@ -39,9 +40,17 @@ export class IHEGatewayV2LambdasNestedStack extends NestedStack {
       versioned: true,
     });
 
+    const iheXcpdResponsesBucket = new s3.Bucket(this, "IHEResponsesBucket", {
```

Comment:
having just dealt with this, i am in favor of creating buckets manually in the console and then referencing them in the stack instead of creating them in the stack. If you ever want to move the bucket or do anything you run into very annoying "this bucket already exists" errors

Reply:
This will likely happen w/ any resource marked as to not be destroyed upon removal, which is the case w/ S3 buckets, RDS DBs, and other resources.

I think we create the bucket w/ CDK, but we need to look into the process to move stuff around if we really need to.

Reply:
Will leave it.